rocprim/include/rocprim/device/config_types.hpp

```diff
@@ -157,6 +165,7 @@ enum class target_arch : unsigned int
     gfx908  = 908,
     gfx90a  = 910,
     gfx1030 = 1030,
+    gfx1102 = 1102,
```

Comment:
I'm assuming because StreamHPC doesn't have access to a Navi31 and Navi32 that we don't have configs for those yet?  If that's the case, then I can try running the autotuning with the the Navi31/32s we have at AMD in put in the configs in a future PR.

Reply:
Yes, that is the case and that would be much appreciated.